modules/apps/commerce/commerce-address-web/src/main/java/com/liferay/commerce/address/web/internal/portlet/action/ActionHelper.java

```diff
@@ -36,54 +36,50 @@
 @Component(enabled = false, service = ActionHelper.class)
 public class ActionHelper {
 
-	public List<CommerceCountry> getCommerceCountries(
-			PortletRequest portletRequest)
+	public List<Country> getCommerceCountries(PortletRequest portletRequest)
```

Comment:
We may want to rename these methods, I can imagine Brian asking for that since they seem to follow the type.